internal/goeland/filters/filter.go

```diff
@@ -154,8 +155,14 @@ func filterDigestGeneric(source *goeland.Source, level int, useFirstEntryTitle b
 	if useFirstEntryTitle && len(source.Entries) > 0 {
 		digest.Title = source.Entries[0].Title
 	}
+	previousSourceTitle := ""
 	content := ""
 	for _, entry := range source.Entries {
+		if entry.IncludeSourceTitle && previousSourceTitle != entry.Source.Title  {
+			content += "<hr/>"

```

Comment:
I wouldn't add <hr/> here. I think it's best to have another filter "includehr", so it can be used either with includelink or includesourcetitle.

Also, I guess we will have to CSS skin it, but I can do it afterwards.

Reply:
Removed the `<hr>` completely as the same result can be achieved by styling it with css see 902b28f